packages/ui/src/app/components/FMBanner.tsx

```diff
@@ -27,24 +28,29 @@ export const FMBanner = () => {
             Testnet participants{' '}
             <StyledLink href="https://joystream.gitbook.io/testnet-workspace/testnet/testnet-rewards">earn</StyledLink>{' '}
             a combination of our testnet token{' '}
-            <StyledLink href="https://joystream.gitbook.io/testnet-workspace/testnet/usdtjoy">$tJOY</StyledLink>, and
-            allocation of our mainnet token $JOY (which will come to life upon mainnet launch).
+            <StyledLink href="https://joystream.gitbook.io/testnet-workspace/testnet/usdtjoy">
+              ${CurrencyName.integerValue}
+            </StyledLink>
+            , and allocation of our mainnet token ${CurrencyName.integerValue} (which will come to life upon mainnet
+            launch).
           </TextMedium>
           <TextMedium>
-            $tJOY can be <StyledLink href="https://www.joystream.org/cashout/">converted</StyledLink> to BCH, (Bitcoin
-            cash cryptocurrency) immediately, whereas $JOY will be allocated shortly after mainnet launch, assuming the
-            contributor has reached the{' '}
+            ${CurrencyName.integerValue} can be{' '}
+            <StyledLink href="https://www.joystream.org/cashout/">converted</StyledLink> to BCH, (Bitcoin cash
+            cryptocurrency) immediately, whereas ${CurrencyName.integerValue} will be allocated shortly after mainnet
+            launch, assuming the contributor has reached the{' '}
             <StyledLink href="https://joystream.gitbook.io/testnet-workspace/testnet/founding-member-program#stages">
               Founding Member
             </StyledLink>{' '}
-            status, by accruing $15k USD worth of $JOY.
+            status, by accruing $15k USD worth of ${CurrencyName.integerValue}.
           </TextMedium>
           <TextMedium>
-            $tJOY balances have no relationship to mainnet $JOY balances, there will be no conversion of any kind.
+            ${CurrencyName.integerValue} balances have no relationship to mainnet ${CurrencyName.integerValue} balances,
+            there will be no conversion of any kind.
           </TextMedium>
           <TextMedium>
             <USFlag />
-            US residents are not eligible for JOY tokens.
+            US residents are not eligible for {CurrencyName.integerValue} tokens.
           </TextMedium>
         </TextBox>
         <CloseButton onClick={toggle} />
```

Comment:
This banner shouldn't be changed at all because it also explains the difference between `$JOY` and `$tJOY`